### PR TITLE
Allow import grade by csv using participant's institutional id.

### PR DESCRIPTION
**Pull request by dzhuang**
Along with more exception handling when importing csv.

### CODE REVIEW

File: course/grades.py [hunk 1]
```diff
@@ -1033,8 +1065,12 @@ def csv_to_grade_changes(
         gchange = GradeChange()
         gchange.opportunity = grading_opportunity
         try:
-            gchange.participation = find_participant_from_id(
-                    course, row[id_column-1])
+            if attr_type == "email_or_id":
+                gchange.participation = find_participant_from_id(
+                        course, row[attr_column-1])
+            else:
```
Comment (inducer) [inline course/grades.py]: Can you change this logic to be `elif attr_type == "...." else: raise`?

**Comment by inducer** [comment]
Looks good aside from the minor wrinkle I pointed out.

**Comment by dzhuang** [comment]
Updated. Thanks for the suggestions.

**Comment by inducer** [comment]
Looks good. Thanks!

**Pull request merged**